src/main/resources/mesosphere/marathon/api/v2/EventResource_get.md

```diff
@@ -0,0 +1,46 @@
+## GET `/v2/events`
```

Comment:
Recent addition in our procedure: Append `<span class="label label-default">v0.9.0</span>`